File: Devatanu/wk01/4-thu/arrays.js
var days_of_the_week =[
    'monday',
    'tuesday',
    'wednesday',
    'thursday',
    'friday',
    'saturday',
    'sunday'
];

days_of_the_week.pop();
days_of_the_week.reverse();
days_of_the_week.push('sunday');
days_of_the_week.reverse();

var days_of_the_week = [
    [
        'monday',
        'tuesday',
        'wednesday',
        'thursday',
        'friday'
    ],
    [
        'saturday',
        'sunday'
    ]
];

days_of_the_week.pop();

days_of_the_week[0].sort();
console.log(days_of_the_week);

File: Devatanu/wk04/2-tue/arrays.rb
require 'pry'

calendar = [
    'Monday',
    'Tuesday',
    'Wednesday',
    'Thursday',
    'Friday',
    'Saturday',
    'Sunday'
]
puts calendar
puts

calendar.unshift(calendar.pop())
puts calendar
puts

new_calendar = [
    calendar[1..-2],
    [calendar[0],calendar.last]
]
puts new_calendar
puts

new_calendar.pop()
puts new_calendar
puts

puts new_calendar.pop().sort()
puts